LibraBFT/Impl/Consensus/PersistentLivenessStorage/Properties.agda

```diff
@@ -0,0 +1,24 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
+
+   Copyright (c) 2021, Oracle and/or its affiliates.
+   Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
+-}
+
+open import Optics.All
+open import LibraBFT.Prelude
+open import LibraBFT.Base.Types
+open import LibraBFT.ImplShared.Base.Types
+open import LibraBFT.ImplShared.Consensus.Types
+open import LibraBFT.ImplShared.Util.Util
+open import LibraBFT.Impl.Consensus.PersistentLivenessStorage
+
+module LibraBFT.Impl.Consensus.PersistentLivenessStorage.Properties where
+
+module saveVoteMSpec (vote : Vote) where
+  -- TODO-2: This contract needs refining (after `saveVoteM` is implemented)
+  postulate
+    contract
+      : ∀ P pre
+        → P (inj₁ fakeErr) pre []
```

Comment:
Even for functions with no logging, I think we should get away from this style of property, which would break if we added logging or any other outputs we can ignore for the purposes of correctness.  Would it be sufficient to say something like:
```
∀ P pre outs
        → NoMsgOut outs
        → P (inj₁ fakeErr) pre outs
```
?